docs/user/cassandra/heatmaps.rst

```diff
@@ -0,0 +1,10 @@
+Cassandra Heatmaps
+==================
+
+Cassandra can generate client-side heatmaps in GeoServer by defining a style and using the built in ``gs:Heatmap``
+process within GeoServer.
+
+To start, add a new SLD style to GeoServer named "heatmap" and use this sld
+:download:`clientside-heatmap.sld <_static/sld/clientside-heatmap.sld>`
```

Comment:
This link is relative, so this need to be changed to either ``<../_static/sld/clientsite-heatmap.sld>`` or ``</user/_static/sld/clientside-heatmap.sld>``.